asab/library/service.py

```diff
@@ -132,13 +132,12 @@ async def _read_disabled(self):
 		# `.disabled.yaml` is read from the first configured library
 		# It is applied on all libraries in the configuration.
 		disabled = await self.Libraries[0].read('/.disabled.yaml')
+		try:
+			self.Disabled = yaml.safe_load(disabled)
+		except Exception:
+			L.exception("Failed to parse '/.disabled.yaml'")
```

Comment:
`disabled` could be `None`, right?

Reply:
1. yes, it is the case when .`disabled.yaml `files do not exist.

There is another case where .`disabled.yaml `exists and the file is empty in that I reckon in this scenario

I should set self.Disabled to {}